typescript/ecs/fargate-service-with-local-image/index.ts

```diff
@@ -0,0 +1,35 @@
+import ec2 = require('@aws-cdk/aws-ec2');
+import ecs = require('@aws-cdk/aws-ecs');
+import cdk = require('@aws-cdk/cdk');
+import path = require('path');
+
+class FargateServiceWithLocalImage extends cdk.Stack {
```

Comment:
For purposes of an example, I wouldn't necessarily wrap this in a class (See: https://github.com/aws-samples/aws-cdk-examples/pull/24#discussion_r271022662)

Reply:
Done